### PR TITLE
Fix typos and whitespace in `worker.py`

### PR DESCRIPTION
**Pull request by hendrikmakait**
Fixes minor typing and whitespace issues I stumbled upon.

### CODE REVIEW

File: distributed/worker.py [hunk 1]
```diff
@@ -287,7 +287,7 @@ class Worker(ServerNode):
     **Volatile State**
 
     These attributes track the progress of tasks that this worker is trying to
-    complete.  In the descriptions below a ``key`` is the name of a task that
```
Comment (jrbourbeau) [inline distributed/worker.py:290]: Just and FYI -- some folks prefer to use two spaces between sentences, so you'll see a lot of this throughout the codebase. We don't have a strict convention around this, so there's a mixture of one and two spaces used today. I don't think there's anything wrong with the changes in this PR, just wanted to provide some additional context

**Review by jrbourbeau — APPROVED** [review]
Thanks @hendrikmakait

**Comment by jrbourbeau** [comment]
rerun tests

File: distributed/worker.py (hunk 1 above)
Comment (hendrikmakait) [reply distributed/worker.py:290]: Thanks for the info, I wasn't aware that this is a convention some folks follow. Good to know!

Comment (mrocklin) [reply distributed/worker.py:290]: I follow it, but pretty much every other english speaker disagrees with me.  I'll probably keep doing it, but won't get in the way of people changing things :)

**Comment by mrocklin** [comment]
This is in.  Thanks @hendrikmakait !

**Pull request merged**